### PR TITLE
Respect case of role name

### PR DESCRIPTION
**Pull request by bersace**
cf. #289 

This **may** break installation relying on ldap2pg lowering case when creating roles.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ follow [merged Pull request
 pages](https://github.com/dalibo/ldap2pg/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged).
 
 
+# Unreleased
+
+**Beware** when upgrading : **ldap2pg will drop roles having uppercase letter in
```
Comment (bersace) [inline docs/changelog.md]: @zurikus, what do you think of this ?

Comment (bersace) [reply docs/changelog.md]: I'm thinking of a way to rename.

Comment (bersace) [reply docs/changelog.md]: Got it. What do you think of this ?

Comment (zurikus) [reply docs/changelog.md]: Yes, I think this is the right note for ldap2pg package users and it should be BOLD.
Truely saying double quotes support not only upper case letters but also "space" and other special characters as well.
http://joxi.ru/brR9x6QI70PGqA

Comment (bersace) [reply docs/changelog.md]: Right about spaces too, thanks for the hint.

**Pull request merged**